packages/quantic/force-app/main/default/lwc/quanticCaseAssistInterface/quanticCaseAssistInterface.js

```diff
@@ -70,7 +83,13 @@ export default class QuanticCaseAssistInterface extends LightningElement {
     this.engine = engine;
     this.actions = {
       ...CoveoHeadlessCaseAssist.loadCaseAssistAnalyticsActions(engine),
+      ...CoveoHeadlessCaseAssist.loadCaseFieldActions(engine),
     };
-    this.engine.dispatch(this.actions.logCaseStart());
+    if (!this.preventLogCaseStartOnInit) {
+      this.engine.dispatch(this.actions.logCaseStart());
+    }
+    if (!this.preventFetchClassificationOnInit) {
+      this.engine.dispatch(this.actions.fetchCaseClassifications());
```

Comment:
I think classifications should not be fetched from the interface component. Classifications are meant to be fetched by the case input controller.

Reply:
Fixed 👌